rllib/utils/__init__.py

```diff
@@ -7,8 +7,6 @@
     renamed_class, renamed_function
 from ray.rllib.utils.filter_manager import FilterManager
 from ray.rllib.utils.filter import Filter
-from ray.rllib.utils.numpy import sigmoid, softmax, relu, one_hot, fc, lstm, \
```

Comment:
Had to take these out due to circular dependencies (these are too specific to be here anyways).